.travis.yml

```diff
@@ -32,13 +32,12 @@ before_install:
 
 install:
   - stack update
-  - travis_wait 120 stack --no-terminal --install-ghc test --only-dependencies
+  - travis_wait 120 stack --no-terminal --install-ghc test --only-dependencies --coverage
 
 script:
   - mkdir .output
-  - stack --no-terminal test --coverage
-  - stack exec grin -- grin/grin/sum_simple.grin
+  - stack test grin:grin-end-to-end-test grin:grin-test --coverage
```

Comment:
This is kind of black magic to me. Could you please explain briefly what this line does, or just give me a pointer where to look?

Reply:
See:
https://github.com/commercialhaskell/stack/issues/731#issuecomment-129081132
https://docs.haskellstack.org/en/stable/build_command/#target-syntax
Basically this is a the way if you have more than one test suite defined in your cabal file to run them. Otherwise it run only one for me.

Reply:
I see. Thank you.